pkg/application/application.go

```diff
@@ -12,8 +12,8 @@ import (
 	"github.com/redhat-developer/odo/pkg/project"
 )
 
-// getDefaultAppName returns application name to be used as a default name in the case where users doesn't provide a name
-// In future this function should generate name with uniq suffix (app-xy1h), because there might be multiple applications.
+// getDefaultAppName returns application name to be used as a default name in case of user doesn't provide a name.
```

Comment:
I guess it should be "... in case the user doesn't provide"

Reply:
Thankyou @mik-dass for reviewing. Yes you right, "USERS" should be replaced by "USER".
plus here i made it to "in case of" which means what should be done in case of something might happen(possibility "user doesn't provide file name"). 
here is the reference.
http://englishstudypage.com/grammar/in-case-of-vs-in-the-case-of-in-english/

Reply:
@acgoliyan Thanks for contributing :+1:  But I was suggesting **in case the user** not **in the case of**. I think **in case** should be used here because http://speakspeak.com/resources/english-grammar-rules/various-grammar-rules/in-case-in-case-of

Reply:
@mik-dass Sorry i misunderstood. You are absolutely right "in case" is right to use here.

Reply:
@acgoliyan No problem thanks again for the contribution :+1: